tests/NHSD.BuyingCatalogue.Organisations.Api.UnitTests/Builders/OrganisationBuilder.cs

```diff
@@ -78,6 +92,28 @@ internal Organisation Build()
                 Address = address,
                 LastUpdated = lastUpdated,
             };
+
+            if (relatedOrganisation != null)
+            {
+                org.RelatedOrganisations.Add(relatedOrganisation);
+            }
+
+            return org;
+        }
+
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.OrderingRules", "SA1204:Static elements should appear before instance elements", Justification = "This Doesn't Actually need to be at the top")]
+        internal static Organisation BuildRelatedOrganisation(Guid index)
```

Comment:
This can be `private`, so the suppression can be removed.

Also, occasionally we might suppress an FxCop warning but we generally don't suppress style warnings.